modAionImpl/src/org/aion/zero/impl/sync/TaskGetStatus.java

```diff
@@ -76,8 +76,7 @@ public void run() {
             try {
                 for (int id : ids) {
                     p2p.send(id, reqStatus);
-
-                    Thread.sleep(Math.max(intervalMin, intervalTotal / ids.size()));
+                    Thread.sleep(1000L);
```

Comment:
This could potentially affect the syncing speed and node last message timestamp.

Reply:
I will be looking into refactoring sync in a separate target, this can be reverted if necessary